src/instructlab/sdg/pipelines/schema/v1.json

```diff
@@ -0,0 +1,151 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["version", "blocks"],
+  "properties": {
+    "version": {
+      "type": "string"
+    },
+    "blocks": {
+      "type": "array",
+      "additionalProperties": false,
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["name", "type", "config"],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "type": {
+            "type": "string"
+          },
+          "drop_duplicates": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "drop_columns": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "gen_kwargs": {
+            "type": "object",
+            "properties": {
```

Comment:
`model` is also supported here - this is how you can point to a non-default model

Reply:
Hmm, this should really be an LLMBlock thing - it's only used there, and it's really arguments for the OpenAI `completions.create()` call

Filed as #138

Reply:
I've added model now